code/procs/jobprocs.dm

```diff
@@ -618,8 +618,9 @@
 
 	var/T = pick(trinket_safelist)
 	var/obj/item/trinket = null
-
-	if (src.traitHolder && src.traitHolder.hasTrait("pawnstar"))
+	if (src.traitHolder && src.traitHolder.hasTrait("plasmatoid"))
+		trinket = new/obj/item/tank/emergency_plasma(src)
+	else if (src.traitHolder && src.traitHolder.hasTrait("pawnstar"))
```

Comment:
honestly all "trinket" traits should be mutually exclusive (but yes being below pawnstar and above everything else is best, so that they only suffer from no breathing if they take pawnstar)